app/views/admin/deals/_deal.html.erb

```diff
@@ -0,0 +1,35 @@
+<div id="<%= dom_id deal %>">
+  <u><%= link_to "#{deal.title}", admin_deal_path(deal) %></u>
+  </br>
+
+  <div>
+    <% deal.deal_images.each do |deal_image| %>
+      <%= image_tag(url_for(deal_image.image)) %>
+    <% end %>
+  </div>
+
+  <div>
+    <strong>Description </strong>
+    <%= deal.description %>
+  </div>
+
+  <div>
+    <strong>Quantity</strong>
+    <%= deal.quantity %>
+  </div>
+
+  <div>
+    <strong>Original Price</strong>
+    <s><%= price_on_deal(deal.price_in_cents) %></s>
+  </div>
+
+  <div>
+    <strong>Deal Price</strong>
+    <%= price_on_deal(deal.discount_price_in_cents) %>
+  </div>
+
+  <div>
+    <strong>Publishable: </strong>
+    <%= deal.assign_attributes(published_at: Time.current); deal.valid? ? "YES" : "NO" %>
```

Comment:
Published at being set means that a deal is published. So, using that seems not right.
instead of using published_at as the deciding factor whether to run those validations or not. Use custom context instead.

Reply:
done